packages/hash/frontend/src/pages/[account-slug]/types/entity-type/entity-type-tabs.tsx

```diff
@@ -78,7 +82,9 @@ export const EntityTypeTabs = () => {
 
         <TabLink
           value="create"
-          href="#"
+          href={`/@${
+            authenticatedUser?.shortname
```

Comment:
Same comment as elsewhere, this is assuming the entity should be created in the entity's namespace and will need to be fixed either now or soon (not sure if we know whether the user is a member of the org being looked at)

Reply:
Apparently org memberships can be seen on the user record received via the `useAuthenticatedUser` hook